DFC.Digital/DFC.Digital.Web.Sitefinity.Core/Startup.cs

```diff
@@ -111,6 +115,8 @@ private static void RegisterRoutes(RouteCollection routes)
 
             routes.MapRoute("serviceStatus", "health/{controller}/{action}", new { controller = "ServiceStatus", action = "Index", id = string.Empty });
             routes.MapRoute("restartsitefinity", "restartsitefinity/{controller}/{action}", new { controller = "AdminPanel", action = "RestartSitefinity", id = string.Empty });
+
+            //routes.MapRoute("dfcapi", "dfcapi/{controller}/{id}", new { id = RouteParameter.Optional });
```

Comment:
do we need to remove this code as it has been moved into another assembly in the install method